thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/dashboard/resources/AnomalyFlattenResource.java

```diff
@@ -0,0 +1,119 @@
+package org.apache.pinot.thirdeye.dashboard.resources;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Preconditions;
+import io.swagger.annotations.ApiOperation;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.pinot.thirdeye.anomalydetection.context.AnomalyFeedback;
+import org.apache.pinot.thirdeye.common.dimension.DimensionMap;
+import org.apache.pinot.thirdeye.datalayer.bao.MergedAnomalyResultManager;
+import org.apache.pinot.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
+import org.apache.pinot.thirdeye.datasource.DAORegistry;
+
+
+@Path("thirdeye/table")
+public class AnomalyFlattenResource {
```

Comment:
Can you add more description?
Is it specific for the ML availability report or it is a general purpose endpoint?

Reply:
It is specific for ML availability report. For the implementation, I tried to make it general to different use cases.

Reply:
Will add the comment in the next update.